tests/core_tests/startup_tests.py

```diff
@@ -15,7 +15,7 @@
 from everest.framework import Module, RuntimeSession
 
 
-@pytest.mark.skip(reason="Fix the config for qwello")
+@pytest.mark.everest_core_config('qwello-test-config-sil.yaml')
```

Comment:
should rather be config-sil-qwello-test to be consistent with the others

Reply:
Changed.